File: src/main/java/no/nav/pto/veilarbportefolje/oppfolging/response/Veilarbportefoljeinfo.java
package no.nav.pto.veilarbportefolje.oppfolging.response;

import lombok.Data;
import lombok.experimental.Accessors;
import no.nav.common.types.identer.AktorId;
import no.nav.common.types.identer.NavIdent;

import java.time.ZonedDateTime;

@Data
@Accessors(chain = true)
public class Veilarbportefoljeinfo {
    private AktorId aktorId;

    private NavIdent veilederId;
    private boolean erUnderOppfolging;
    private boolean nyForVeileder;
    private boolean erManuell;
    private ZonedDateTime startDato;
}

File: src/main/java/no/nav/pto/veilarbportefolje/oppfolging/domene/Veilarbportefoljeinfo.kt
package no.nav.pto.veilarbportefolje.oppfolging.domene

import no.nav.common.types.identer.AktorId
import no.nav.common.types.identer.NavIdent
import java.time.ZonedDateTime

data class Veilarbportefoljeinfo(
    val aktorId: AktorId? = null,
    val veilederId: NavIdent? = null,
    val erUnderOppfolging: Boolean = false,
    val nyForVeileder: Boolean = false,
    val erManuell: Boolean = false,
    val startDato: ZonedDateTime? = null,
    val tilordnetTidspunkt: ZonedDateTime? = null,
)
